manifests/host-pod.yaml.j2

```diff
@@ -14,9 +14,9 @@ spec:
   containers:
   - name: {{ pod_name }}
     image: {{ test_image }}
-    command:
-      - "{{ command }}"
-    args: {{ args }}
+    command: {{ command }}
```

Comment:
Does this fix an issue? Or just a style change?

Reply:
This change is not required anymore after merging with #95